source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -103,6 +104,22 @@ std::vector<std::string> initNamespaces(const Protobuf::RepeatedPtrField<std::st
   return namespaces;
 }
 
+absl::optional<envoy::config::core::v3::TypedExtensionConfig>
```

Comment:
nit: I think the return type could be OptRef<envoy::config::core::v3::TypedExtensionConfig>